src/main/java/com/meilisearch/sdk/SearchRequest.java

```diff
@@ -300,6 +307,7 @@ private SearchRequest(
             String cropMarker,
             String highlightPreTag,
             String highlightPostTag,
+            String matchingStrategy,
```

Comment:
I think you can use an enum here :)

this may help you: https://www.baeldung.com/java-enum-values

Reply:
I put it on a new `MatchingStrategy ` file inside model directory, but I wonder if I could just put it on the `SearchRequest` file? WDYT?